website/src/layouts/base/header/HamburgerMenu.astro

```diff
@@ -0,0 +1,73 @@
+---
+import { getSession } from 'auth-astro/server';
+
+import HamburgerMenuItem from './HamburgerMenuItem.astro';
+import { getPathogenMegaMenuSections } from './getPathogenMegaMenuSections';
+import { LoginButton } from '../../../components/auth/LoginButton';
+import { Page } from '../../../types/pages';
+
+type Props = {
+    forceLoggedOutState: boolean;
+};
+
+const { forceLoggedOutState } = Astro.props;
+
+const pathogenMegaMenuSections = getPathogenMegaMenuSections();
+
+const session = await getSession(Astro.request);
+const showLoggedInState = !forceLoggedOutState && session?.user !== undefined;
+---
+
+<div>
+    <input id='menu__toggle' class='peer hidden' type='checkbox' />
+    <label
+        for='menu__toggle'
+        class='iconify absolute right-5 top-4 cursor-pointer text-4xl mdi--hamburger-menu peer-checked:mdi--window-close'
+    >
+    </label>
+
+    <ul
+        class='fixed left-full top-0 mt-16 h-[calc(100dvh-4rem)] w-full overflow-scroll border bg-white pl-4 duration-200 peer-checked:left-0'
+    >
+        {
+            pathogenMegaMenuSections.map((section) => (
+                <HamburgerMenuItem
+                    content={{
+                        type: 'section',
+                        navigationEntries: section.navigationEntries,
+                    }}
+                    style='toplevel'
+                >
+                    {section.headline}
+                </HamburgerMenuItem>
+            ))
+        }
+        <HamburgerMenuItem content={{ type: 'item', href: Page.dataSources }} style='toplevel'
+            >Data Sources
+        </HamburgerMenuItem>
+        {
+            showLoggedInState ? (
+                <HamburgerMenuItem
+                    content={{
+                        type: 'section',
+                        navigationEntries: [
+                            { label: session.user!.name! },
```

Comment:
It is a bit strange, that the username has the same style as the others links. But I think this is fine for now, since we do not want to launch with the login anyway.

Reply:
Let's improve that later :+1: